clang/lib/CodeGen/CGCall.cpp

```diff
@@ -5581,6 +5620,13 @@ RValue CodeGenFunction::EmitCall(const CGFunctionInfo &CallInfo,
   // Emit the actual call/invoke instruction.
   llvm::CallBase *CI;
   if (!InvokeDest) {
+    if (CGM.getCodeGenOpts().FPAccuracy) {
+      const auto *FD = dyn_cast_or_null<FunctionDecl>(TargetDecl);
```

Comment:
```suggestion
      const auto *FD = dyn_cast_if_present<FunctionDecl>(TargetDecl);
```
the other form is deprecated.